detox/e2e/support/server_api/setup.js

```diff
@@ -16,22 +15,18 @@ export const apiInit = async ({
     teamOptions = {type: 'O', prefix: 'team'},
     userOptions = {prefix: 'user'},
 } = {}) => {
-    try {
-        const {team} = await Team.apiCreateTeam(teamOptions);
-        const {channel} = await Channel.apiCreateChannel({...channelOptions, teamId: team.id});
-        const {user} = await User.apiCreateUser(userOptions);
+    const {team} = await Team.apiCreateTeam(teamOptions);
+    const {channel} = await Channel.apiCreateChannel({...channelOptions, teamId: team.id});
+    const {user} = await User.apiCreateUser(userOptions);
 
-        await Team.apiAddUserToTeam(user.id, team.id);
-        await Channel.apiAddUserToChannel(user.id, channel.id);
+    await Team.apiAddUserToTeam(user.id, team.id);
+    await Channel.apiAddUserToChannel(user.id, channel.id);
 
-        return {
-            channel,
-            team,
-            user,
-        };
-    } catch (err) {
-        return getResponseFromError(err);
```

Comment:
`getResponseFromError` is handled separately by each API request, so this is redundant here.